doc/release-notes.md

```diff
@@ -1,6 +1,240 @@
 # release notes
 
 ## current release
+- release date: 2023-04-XX
+- status: available
+- overview of changes (See [PR XXX](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/) for additional details):
+  - This release adds the following data: 
+    - 744 methylation array data samples
+      - 728 PBTA samples (17 normal, 711 tumor)
+      - 16 TARGET samples (6 normal, 10 tumor)
+    - 1264 RNA-seq samples
+      - 1264 PBTA samples (3 normal, 1261 tumor)
+    - 2759 WGS/WXS samples
+      - 3 Kentucky samples (3 normal)
+      - 2441 PBTA WGS samples (1171 normal, 1270 tumor)
+      - 315 PBTA WXS samples (160 normal, 155 tumor)
+  
+  - files changed
+    - files added
+      - `20038D-17Q6-01.regions.100bp_padded.bed`
+      - `S0274956_Padded_HG38.merged.bed`
+      - `agilent-v4-targets-ucsc.100bp_padded.bed`
+      - `UCSC_hg19-GRCh37_Ensembl2RefSeq.tsv`
+      - `cnv-gatk.seg.gz`
+      - `fusion-annoFuse.tsv.gz`
+      - `fusion_summary_lgg_hgg_foi.tsv`
+      - `independent-specimens.methyl.primary-plus.eachcohort.tsv`
+      - `independent-specimens.methyl.primary.eachcohort.tsv`
+      - `independent-specimens.methyl.relapse.eachcohort.tsv`
+      - `methyl-md5sum.txt`
+      - `onco1500-v2-targets-ucsc.100bp_padded.bed`
+      - `onco1500-v4-targets-ucsc.100bp_padded.bed`
+      - `onco1500-v6-targets-ucsc.100bp_padded.bed`
+      - `onco1500-v6a-targets-ucsc.100bp_padded.bed`
+    - files renamed
+      - `biospecimen_id_to_bed_map.txt` renamed to `biospecimen_id_to_bed_map.tsv`
+      - `independent-specimens.rnaseqpanel.primary-plus.eachcohort.tsv` renamed to `independent-specimens.rnaseq.primary-plus-pre-release.tsv`
+      - `independent-specimens.rnaseqpanel.primary.pre-release.tsv` renamed to `independent-specimens.rnaseq.primary-pre-release.tsv`
+      - `independent-specimens.rnaseqpanel.relapse.pre-release.tsv` to `independent-specimens.rnaseq.relapse-pre-release.tsv`
+    - files removed or no longer automatically downloaded
+      - `fusion_summary_lgat_foi.tsv`
+      - `methyl-beta-values.rds`
+      - `methyl-m-values.rds`
+      - `tcga-gene-counts-rsem-expected_count-collapsed.rds`
+ 
+  - Updated data to use GENCODE v39 GDC
```

Comment:
Can we create a MAJOR UPDATES section at the top, which summarizes new data added and then also mentions realignment of selected RNA-Seq (PBTA, GKMF, TARGET) to GENCODE v39 and DNA re-annotated with VEP 105 (GENCODE v39 equivalent).